pkg/broker/binding_operations_test.go

```diff
@@ -105,7 +105,7 @@ func TestBindAlreadyExisting(t *testing.T) {
 		ServiceID: testServiceID,
 	}, true)
 
-	assert.EqualError(t, err, apiresponses.ErrBindingAlreadyExists.Error())
+	assert.Nil(t, err)
```

Comment:
This change implies that, given the name of the function, if a Bind Already Exists, there will be no error. Is that correct? am I missing something?

Reply:
Yes that is correct, this is temporary though. It's temporary, because the service broker api doesn't have a StatusOK code for dealing with this. According to the specs it should return StatusOK 200 whenever a user is trying to provision an instance with the same instance_id and with the same parameters.